addons/video/src/addon.xml

```diff
@@ -29,5 +29,7 @@
         </property>
         <property name="offlineMessage" nameLabel="video_property_offline_message" type="text" />
 		<property name="enableVideoSpeedController" nameLabel="enable_video_speed_controller" type="boolean" />
+		<property name="Base width" nameLabel="video_property_base_width" type="string" />
+		<property name="Base height" nameLabel="video_property_base_height" type="string" />
```

Comment:
podbijam - type "int" - można wprowadzić litery do inputów:
![image](https://github.com/icplayer/icplayer/assets/87431460/91436c7c-78de-4481-9af1-5a4996709efe)

Reply:
nie ma takiego property jak int. Jeżeli wpiszesz w type "int", "integer", to go nie rozpozna i wykorzysta string, bo to default.